app/lib/shared/handler_helpers.dart

```diff
@@ -136,7 +136,7 @@ shelf.Handler _requestContextWrapper(shelf.Handler handler) {
     if (!context.uiCacheEnabled && !CacheHeaders.hasCacheHeader(rs.headers)) {
```

Comment:
This and the `requestContext.isNotAuthenticated` logic in app/lib/frontend/handlers/headers.dart is a bit weird.

Probably we should only have defaults in one location.

Reply:
But I think we can clean that up later.